<launch>  
  <node pkg="miivii_gmsl_camera" exec="miivii_gmsl_camera_node" name="miivii_gmsl_camera_node" output="screen">  
    <param name="sync_freq" value="30"/>
    <param name="video0.active" value="true"/>
    <param name="video0.camera_res" value="1920x1536"/>
    <param name="video1.active" value="true"/>
    <param name="video1.camera_res" value="1920x1536"/>
    <param name="video2.active" value="true"/>
    <param name="video2.camera_res" value="1920x1536"/>
    <param name="video3.active" value="true"/>
    <param name="video3.camera_res" value="1920x1536"/>
    <param name="video4.active" value="true"/>
    <param name="video4.camera_res" value="1920x1536"/>
    <param name="video5.active" value="true"/>
    <param name="video5.camera_res" value="1920x1536"/>
    <param name="video6.active" value="true"/>
    <param name="video6.camera_res" value="1920x1536"/>
    <param name="video7.active" value="true"/>
    <param name="video7.camera_res" value="1920x1536"/>
  </node>
</launch>
